src/ui/actions/network.ts

```diff
@@ -126,3 +128,38 @@ export function selectAndFetchRequest(requestId: RequestId): UIThunkAction {
     });
   };
 }
+
+export function seekToRequestFrame(
+  requestId: RequestId,
+  frameId: string,
+  cx: Context
+): UIThunkAction {
+  return async (dispatch, getState, { ThreadFront }) => {
+    const state = getState();
+    const request = state.network.requests.find(request => request.id === requestId);
+    const loadedRegions = getLoadedRegions(state);
+
+    // Don't select a request that's not within a loaded region.
+    if (!request || !loadedRegions || !isPointInRegions(loadedRegions.loaded, request.point)) {
+      return;
+    }
+
+    const requestSummary = getSummaryById(state, requestId);
+    if (!requestSummary) {
+      console.error(`Could not find summary for request "${requestId}"`);
+      return;
+    }
+
+    const timeStampedPoint = requestSummary.point;
+    const pause = ThreadFront.ensurePause(timeStampedPoint.point, timeStampedPoint.time);
+    ThreadFront.timeWarpToPause(pause);
```

Comment:
I'd expect this to fail because `pause.hasFrames` isn't set yet: https://github.com/replayio/devtools/blob/39e705382386b7460bc115e679c8ca4f9667a005/packages/protocol/thread/thread.ts#L350-L357
I'd use `ThreadFront.timeWarp(point, time, true)` followed by `ThreadFront.getFrames()` instead of `ThreadFront.ensurePause(point, time)` and then `ThreadFront.timeWarpToPause(pause)` followed by `pause.getFrames()`